ci-generate-start-node

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -ev
+
+START_NODE_DIR=$HOME/bin
+
+case $(uname -s) in
+  *Linux) SHELL="/bin/bash"
+  ;;
+  *BSD) SHELL="/bin/tsch"
+esac
+
+mkdir -p $START_NODE_DIR
+cat $HOME/ci-tools/start_node.in | sed \
+    -e "s^@SHELL@^$SHELL^g" \
```

Comment:
Not sure if it´s possible due to different shells around, but can we use the "standard" convention from other sed invocation?

``
"s/@SHELL@/$SHELL/g" for example ?
``

^ can be used to catch beginning of lines and stuff and might be confusing.

Reply:
no. Because $SHELL contains a path and sed thinks that is part of the delimiter (it's the first thing I tried!). ^ isn't maybe ideal but it seemed an unlikely character in a shell name